Emby.Server.Implementations/ConfigurationOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Emby.Server.Implementations
+{
+    public static class ConfigurationOptions
+    {
+        public static readonly Dictionary<string, string> Configuration = new Dictionary<string, string>
+        {
+            {"ManagedFileSystem:DefaultDirectory", null},
+            {"ManagedFileSystem:EnableSeparateFileAndDirectoryQueries", "True"}
```

Comment:
Do we need this option at all? Some old comments said it had to be tweaked for Mono on Linux, but we had it hardcoded to `true` since we are on .net core. I suggest removing this option.

Reply:
Both these options are unnecessary actually. Removed them both.

Program.cs and ApplicationHost are both getting really large, the extra class let's us configure options for ServerImplementations services that only devs can toggle and not users while still providing a centralised place for them. With this implementation, if we want to let the user configure a specific option, it's as simple as removing it from the ConfigurationOptions dictionary and adding it to an external configurations file that also get''s added to the IConfiguration.